projects/CO-DETR/configs/codino/co_dino_5scale_r50_lsj_8xb2_3x_coco.py

```diff
@@ -1,4 +1,4 @@
 _base_ = ['co_dino_5scale_r50_lsj_8xb2_1x_coco.py']
 
-param_scheduler = [dict(milestones=[30])]
+param_scheduler = [dict(type='MultiStepLR', milestones=[30])]
```

Comment:
Maybe it would be better to do it like in base?

https://github.com/open-mmlab/mmdetection/blob/main/projects/CO-DETR/configs/codino/co_dino_5scale_r50_lsj_8xb2_1x_coco.py#L335
```py
max_epochs = 12
train_cfg = dict(max_epochs=max_epochs)

param_scheduler = [
    dict(
        type='MultiStepLR',
        begin=0,# Defaults to 0.
        end=max_epochs, # Defaults to INF.
        by_epoch=True, # Defaults to True
        milestones=[30],
        gamma=0.1) # gamma=0.1 - default
]
```